readme.md

```diff
@@ -94,7 +109,7 @@ How is the concept of OAuth related to a valet key?
 
 Your Answer:
 ```text
-
+When you valet park, you hand off your private keys to someone to park your precious car for you instead of doing it yourself. OAuth is when you hand off your data and resources to a third party so that they can authorize a user for you instead of having them sign in locally from your app. OAuth and the valet are both great for convenience and speed.  
 ```
```

Comment:
While these things are true, it relates to a valet key in that valet key's only have access to certain parts of a car. Similarly, OAuth allows you to share your account on a 3rd party provider (e.g. Facebook, Twitter, Google), without given the consumer app your password, or necessarily full access to your account, such as reading / modifying  your data, posting on your behalf, etc.